tests/integration/dump_test.go

```diff
@@ -98,7 +98,58 @@ func Test_Dump_SkipConsumers(t *testing.T) {
 	}
 	for _, tc := range tests {
 		t.Run(tc.name, func(t *testing.T) {
-			runWhen(t, "enterprise", ">=3.2.0")
+			runWhen(t, "enterprise", ">=3.2.0 <3.4.0")
+			teardown := setup(t)
+			defer teardown(t)
+
+			assert.NoError(t, sync(tc.stateFile))
+
+			var (
+				output string
+				err    error
+			)
+			if tc.skipConsumers {
+				output, err = dump(
+					"--skip-consumers",
+					"-o", "-",
+				)
+			} else {
+				output, err = dump(
+					"-o", "-",
+				)
+			}
+			assert.NoError(t, err)
+
+			expected, err := readFile(tc.expectedFile)
+			assert.NoError(t, err)
+			assert.Equal(t, output, expected)
+		})
+	}
+}
+
+func Test_Dump_SkipConsumers_After340(t *testing.T) {
+	tests := []struct {
+		name          string
+		stateFile     string
+		expectedFile  string
+		skipConsumers bool
+	}{
+		{
+			name:          "dump with skip-consumers",
+			stateFile:     "testdata/dump/002-skip-consumers/kong34.yaml",
+			expectedFile:  "testdata/dump/002-skip-consumers/expected.yaml",
+			skipConsumers: true,
+		},
+		{
+			name:          "dump with no skip-consumers",
+			stateFile:     "testdata/dump/002-skip-consumers/kong34.yaml",
+			expectedFile:  "testdata/dump/002-skip-consumers/expected-no-skip-34.yaml",
+			skipConsumers: false,
+		},
+	}
+	for _, tc := range tests {
+		t.Run(tc.name, func(t *testing.T) {
+			runWhen(t, "enterprise", ">=3.4.0")
```

Comment:
Couldn't this be run against Konnect as well?

Reply:
Totally, added. Thanks for catching!